includes/discovery/bgp-peers/firebrick.inc.php

```diff
@@ -33,17 +33,17 @@
     if (strlen($address) > 15) {
         $address = IP::fromHexString($address)->compressed();
     }
-
-    if (isset($value['fbBgpPeerTableId'])) {
+    if (isset($value['fbBgpPeerTableId']) && $value['fbBgpPeerTableId'] !== '') {
         $bgpPeers[$value['fbBgpPeerTableId']][$address] = $value;
     } else {
-        $bgpPeers[0][$address] = $value;
+        $bgpPeers[null][$address] = $value;
```

Comment:
$bgpPeers[null][whatever] does not make sense. so this part of the fix is for sure wrong.

Reply:
The idea here is that null can be stored in the DB (and means global routing table) but cannot be stored in a hash table as a key.

Reply:
We now use 0 in the hash table but null in the DB (and convert accordingly).